### ISSUE TITLE
Not all deleted items are logged in event log?

### ISSUE THREAD

**Issue by jikquantopian** [bitwarden/core#422]
I just created a secure note in our organization vault and then deleted it shortly afterward. There is a "Created item" log entry in the event log for the event, but no corresponding "Deleted item" log entry.

Then I created a login and deleted it, and both the creation and deletion were logged.

Then I created another secure note and deleted it, and both the creation and deletion were logged.

I do not understand why there was no event logged for deleting the first item I created.

In addition, it appears that one of my coworkers deleted something like eight items all at once earlier today, and none of those deletions were logged.

Here are the most recent event log entries for our organization:

![image](https://user-images.githubusercontent.com/41483888/51002972-fec5bb00-1502-11e9-8f8b-c75def1fbd85.png)

And here is what I see when I attempt to search for the first of those:

![image](https://user-images.githubusercontent.com/41483888/51003001-18670280-1503-11e9-8d9a-e469c25845ac.png)

In short, it ain't there, but its deletion was never logged.

**Comment by kspearrin** [comment]
How are you deleting the item? I was able to find that doing bulk deletes from the web vault does not log an event, which is an issue. Every other way I delete an item properly logs the event.

**Comment by jikquantopian** [comment]
I think perhaps the first time I deleted the secure note I did it by checking the box to the left of it in the item list and then clicking the gear and selecting "Delete items". Is that what you mean by "bulk delete"? If so, then it sounds like the issue you found is what I experienced, and perhaps what my coworker experienced earlier today as well.

**Comment by kspearrin** [comment]
Yes, the API call and server logic path for the single delete button is different than the one used for the checkbox delete (bulk delete). This should be fixed in next release with https://github.com/bitwarden/core/commit/7116072cfacbed45a5d787a7c2e73f557b857c08
